test/test-project/src/test/editor/editorView-test.ts

```diff
@@ -149,4 +162,81 @@ describe('EditorView', () => {
             expect(titles).deep.equals(allTitles);
         });
     });
-});
\ No newline at end of file
+});
+
+async function hasTab(title: string): Promise<boolean> {
```

Comment:
don't you think that this could be part of EditorView? I think it is pitty to have these kind of methods located only in test classes...

Reply:
I have introduced EditorTabNotFound to fulfil this use case.